interactive_textinput_printer.go

```diff
@@ -226,6 +239,10 @@ func (p InteractiveTextInputPrinter) updateArea(area *cursor.Area) string {
 		}
 	}
 
+	av := internal.GetStringMaxWidth(areaText)
+	vv := internal.GetStringMaxWidth(p.text)
+
+	_ = av + vv
```

Comment:
Just curious what this is doing? Does it not update unless you get the max widths?

Reply:
Oops :D, I was testing something, and these are only debug statements.